cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,79 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
+  const userName = 'iravakula';
```

Comment:
Avoid storing credentials directly in the code. It's better to store them in environment variables and access them using 'Cypress.env'.